allennlp/commands/checklist.py

```diff
@@ -0,0 +1,195 @@
+"""
+The `checklist` subcommand allows you to sanity check your
+model's predictions using a trained model and its
+[`Predictor`](../predictors/predictor.md#predictor) wrapper.
+"""
+
+from typing import Optional, Dict, Any, List
+import argparse
+import sys
+import json
+
+from overrides import overrides
+
+from allennlp.commands.subcommand import Subcommand
+from allennlp.common.checks import check_for_gpu
+from allennlp.models.archival import load_archive
+from allennlp.predictors.predictor import Predictor
+from allennlp.sanity_checks.task_checklists.task_suite import TaskSuite
+
+
+@Subcommand.register("checklist")
+class CheckList(Subcommand):
+    @overrides
+    def add_subparser(self, parser: argparse._SubParsersAction) -> argparse.ArgumentParser:
+
+        description = """Run the specified model through a checklist suite."""
+        subparser = parser.add_parser(
+            self.name,
+            description=description,
+            help="Run a trained model through a checklist suite.",
+        )
+
+        subparser.add_argument(
+            "archive_file", type=str, help="the archived model to make predictions with"
+        )
+
+        subparser.add_argument("task", type=str, help="the name of the task suite")
+
+        subparser.add_argument("--checklist-suite", type=str, help="the checklist suite path")
```

Comment:
Where do we find these? Is this https://github.com/marcotcr/checklist/tree/master/release_suites?

Reply:
We will add our default suites to cloud. And people can also write their own suites.